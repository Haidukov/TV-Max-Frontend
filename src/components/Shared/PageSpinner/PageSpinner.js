import React from 'react';
import ClipLoader from 'react-spinners/ClipLoader';

import s from './PageSpinner.scss';

const PageSpinner = () => (
  <div className={s.spinner} role='status'>
    <ClipLoader color='black' loading size={100} />
  </div>
);

export default PageSpinner;
