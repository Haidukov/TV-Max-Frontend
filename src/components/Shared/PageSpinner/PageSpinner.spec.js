import React from 'react';
import { render } from '@testing-library/react';
import PageSpinner from './PageSpinner';

describe('<PageSpinner />', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<PageSpinner />);
    expect(asFragment()).toMatchSnapshot();
  });
});
