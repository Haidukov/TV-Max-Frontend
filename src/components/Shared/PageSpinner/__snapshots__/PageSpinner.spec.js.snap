// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageSpinner /> should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="spinner"
    role="status"
  >
    <span
      class="css-1k4jg2i"
    />
  </div>
</DocumentFragment>
`;
